sdk/src/claim.rs

```diff
@@ -1976,57 +1976,29 @@ impl Claim {
             return Err(Error::ValidationRule("No Action array in Actions".into()));
         }
 
-        // check Claim.v2 first action rules
-        let first_actions_assertion = if claim.version() > 1 {
-            // check created actions first
-            let mut found_first_action = None;
-
-            if let Some(assertion) = created_actions.first() {
-                let first_actions = Actions::from_assertion(assertion.assertion())?;
-                let first_actions_first_action = &first_actions.actions()[0];
-
-                if first_actions_first_action.action() == c2pa_action::OPENED
-                    || first_actions_first_action.action() == c2pa_action::CREATED
-                {
-                    found_first_action = Some(assertion);
-                }
+        // Skip further checks for v1 claims if not in strict validation mode
+        if claim.version() == 1 {
+            if let Ok(false) = get_settings_value::<bool>("verify.strict_v1_validation") {
```

Comment:
I guess your reasoning behind checking these two arrays vs the actions array is that v1 claims won't have any assertions here. But I'm curious about why we need to support v1 strict validation at all.